gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/statistics/functions/AddSumDegreesToGraphHeadCrossFunction.java

```diff
@@ -40,13 +40,6 @@ public AddSumDegreesToGraphHeadCrossFunction(String propertyKey) {
     this.propertyKey = propertyKey;
   }
 
-  /**
```

Comment:
I think this explanation is needed here because there is no good description of the parameters in the parent class